src/parser/jobs/rpr/modules/OtherGauges.tsx

```diff
@@ -114,10 +116,6 @@ export class OtherGauges extends CoreGauge {
 			this.shroudGauge.spend(HIGH_SHROUD_MOD)
 			break
 
-		case this.data.actions.PLENTIFUL_HARVEST.id:
```

Comment:
Correct me if I'm wrong, but Arcane Circle grants a "free use of Enshroud" status now, right? We should account for that in here as well.

Reply:
You are correct. 
![image](https://github.com/user-attachments/assets/282cec90-255c-4155-910f-2efb41fd7cfd)
I've added a check similar to reawaken on Viper for Enshroud consuming gauge.